pkg/plugin/plugin.go

```diff
@@ -18,9 +18,49 @@ const (
 	MethodSendNotification = "SendNotification"
 )
 
+// FormElement describe a form element required for the channel config
+type FormElement struct {
+	// Element name
+	Name string `json:"name"`
+	// Element type: string = text, bool = checkbox, text = textarea, option = select, options = select[multiple], secret = password
+	Type string `json:"type"`
+	// Element label map. Locale in the standard format (language_REGION) as key and corresponding label as value.
+	//
+	// e.g. {"en_US": "Save", "de_DE": "Speichern"}
+	Label map[string]string `json:"label"`
+	// Element placeholder
+	Placeholder string `json:"placeholder,omitempty"`
+	// Element title
+	Help map[string]string `json:"help,omitempty"`
+	// Element default value: bool for checkbox, string for other elements
+	Default any `json:"default,omitempty"`
+	// Set true if element is required, omit otherwise
+	Required bool `json:"required,omitempty"`
+	// Select element options map:  key => value. Only required for type option and options
+	Options map[string]string `json:"options,omitempty"`
+}
+
+// Info contains plugin information.
 type Info struct {
-	Name             string          `json:"display_name"`
-	ConfigAttributes json.RawMessage `json:"config_attrs"`
+	Type             string          `db:"type" json:"type"`
+	Name             string          `db:"name" json:"name"`
+	Version          string          `db:"version" json:"version"`
+	Author           string          `db:"author" json:"author"`
+	ConfigAttributes json.RawMessage `db:"config_attrs" json:"config_attrs"` // FormElement as json
```

Comment:
The comment should at least make clear that's a JSON-encoded list/slice of `FormElement`.